check/test.go

```diff
@@ -135,9 +136,11 @@ func (t *testItem) execute(s string) *testOutput {
 				}
 			}
 
+			expectedResultPattern := ""
 			result.actualResult = strings.ToLower(flagVal)
 			switch t.Compare.Op {
 			case "eq":
+				expectedResultPattern = "'%s' Is equal to '%s'"
```

Comment:
I think all these should be lower case `is` rather than `Is` (same for all the other expected result strings)